lib/Config/Menu.php

```diff
@@ -454,6 +474,10 @@ public function formEditor() {
     $this->container->get(FormEditor::class)->render();
   }
 
+  public function formEditorTemplateSelection() {
+    $this->container->get(FormEditor::class)->renderTemplateSelection();
```

Comment:
I think the page should reside in https://github.com/mailpoet/mailpoet/tree/master/lib/AdminPages/Pages.